src/material.cpp

```diff
@@ -51,8 +51,10 @@ mat_burn_data load_mat_burn_data( JsonObject &jsobj )
     return bd;
 }
 
-void material_type::load( JsonObject &jsobj, const std::string & )
+void material_type::load( JsonObject &jsobj, const std::string &src )
 {
+    bool strict = src == "core";
```

Comment:
Yes, `game::is_strict( const std::string &mod )` would suffice, but not in this PR as I don't want to miss that line when grepping to make the latter change